source/agora/common/EnrollmentManager.d

```diff
@@ -79,8 +80,12 @@ public class EnrollmentManager
         this.db.execute("CREATE TABLE IF NOT EXISTS node_enroll_data " ~
             "(key CHAR(128) PRIMARY KEY, val BLOB NOT NULL)");
 
-        // set node's KeyPair object
+        // create Pair object from KeyPair object
+<<<<<<< HEAD
```

Comment:
Rebase error

Reply:
Thank you, I'm now trying to resolve the error.